images/manageiq-base/Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM registry.access.redhat.com/ubi9/ubi
-MAINTAINER ManageIQ https://manageiq.org
+ARG FROM_REPO=docker.io/manageiq
+ARG FROM_TAG=latest
+
+# alias stage for rpms
+FROM ${FROM_REPO}/manageiq-rpms:${FROM_TAG} as rpms
+
+################################################################################
 
-ARG LOCAL_RPM
+FROM registry.access.redhat.com/ubi9/ubi
+ARG FROM_REPO
+ARG FROM_TAG
```

Comment:
If ARGs are needed in later stages, but are defined in earlier stages, they have to be redeclared in the later stage (but blank).  That's why these are here.